api/api-doc/tasks.json

```diff
@@ -0,0 +1,230 @@
+                     "required":false,
+                     "allowMultiple":false,
+                     "type":"string",
+                     "paramType":"query"
+                  },
+                  {
+                     "name":"flush_memtables",
+                     "description":"Controls flushing of memtables before compaction (true by default). Set to \"false\" to skip automatic flushing of memtables before compaction, e.g. when tables were flushed explicitly before invoking the compaction api.",
+                     "required":false,
+                     "allowMultiple":false,
+                     "type":"boolean",
+                     "paramType":"query"
+                  }
+               ]
+            }
+         ]
+      },
+      {
+         "path":"/tasks/compaction/keyspace_cleanup/{keyspace}",
+         "operations":[
+            {
+               "method":"POST",
+               "summary":"Trigger a cleanup of keys on a single keyspace asynchronously, returns uuid which can be used to check progress with task manager",
+               "type": "string",
+               "nickname":"force_keyspace_cleanup_async",
+               "produces":[
+                  "application/json"
+               ],
+               "parameters":[
+                  {
+                     "name":"keyspace",
+                     "description":"The keyspace to query about",
+                     "required":true,
+                     "allowMultiple":false,
+                     "type":"string",
+                     "paramType":"path"
+                  },
+                  {
+                     "name":"cf",
+                     "description":"Comma-separated column family names",
+                     "required":false,
+                     "allowMultiple":false,
+                     "type":"string",
+                     "paramType":"query"
+                  }
+               ]
+            }
+         ]
+      },
+      {
+         "path":"/tasks/compaction/keyspace_offstrategy_compaction/{keyspace}",
+         "operations":[
+            {
+               "method":"POST",
+               "summary":"Perform offstrategy compaction, if needed, in a single keyspace asynchronously, returns uuid which can be used to check progress with task manager",
+               "type":"string",
+               "nickname":"perform_keyspace_offstrategy_compaction_async",
+               "produces":[
+                  "application/json"
+               ],
+               "parameters":[
+                  {
+                     "name":"keyspace",
+                     "description":"The keyspace to operate on",
+                     "required":true,
+                     "allowMultiple":false,
+                     "type":"string",
+                     "paramType":"path"
+                  },
+                  {
+                     "name":"cf",
+                     "description":"Comma-separated table names",
```

Comment:
nit: for consistency, why not 'Comma-separated column family names' ?

Reply:
Parameters descriptions were copied from non-async api, should they be changed for non-async also?

Aren't we rather changing from "column family" to "table" and not the other way round?

Reply:
Yes, "column family" is the legacy term, but it is used throughout the API. I agree, best be consistent with the sync API w.r.t. parameter names.

Reply:
As you stayed in API V1, Parameter names (cf) should be consistent, the description should be as clear as possible, I would use table with perhaps adding  (Column-family) to make it clear